lib/backend/etcdbk/etcd.go

```diff
@@ -407,6 +408,9 @@ func (b *EtcdBackend) reconnect(ctx context.Context) error {
 		MaxCallSendMsgSize: b.cfg.MaxClientMsgSizeBytes,
 	})
 	if err != nil {
+		if errors.Is(err, context.DeadlineExceeded) {
+			return trace.WrapWithMessage(err, "failed to dial all etcd endpoints: %s", b.nodes)
```

Comment:
It's still not entirely clear that the failure is due to a timeout.

How about this?
```suggestion
			return trace.WrapWithMessage(err, "timed out dialing etcd endpoints: %s", b.nodes)
```

Reply:
Thanks! That's better.